### ISSUE TITLE
[Request] Support for PS4 Firmware 11.52

### ISSUE THREAD

**Issue by JBERNARD024** [GoldHEN/GoldHEN#231]
I wanted to know if a jailbreak for version 11.52 is in development. If not, how long might it take?

**Comment by KrisztianKende** [comment]
Currently the [PPPwn](https://github.com/TheOfficialFloW/PPPwn) is the latest usable exploit. The next exploit is expected to arrive in about a couple of years. Or sooner, no one knows yet, but you asked this question in the wrong place.

**Comment by ad1k4h** [comment]
Any Updates about this?

**Comment by KrisztianKende** [comment]
Yes, just use GoldHEN v2.4b18.5 or later (current version is 2.4b18.10).
